dist/angular-storage.js

```diff
@@ -12,18 +12,25 @@ angular.module('angular-storage',
 
 angular.module('angular-storage.cookieStorage', [])
   .service('cookieStorage', ["$injector", function ($injector) {
-    var $cookieStore = $injector.get('$cookieStore');
+    var $cookies = $injector.get('$cookies');
```

Comment:
why are you renaming $cookieStore to $cookies ?
`$injector.get('$cookies')` is not failing?

https://docs.angularjs.org/api/ngCookies/service/$cookieStore